src/index.js

```diff
@@ -0,0 +1,138 @@
+// import inquirer and fs
+
+const inquirer = require("inquirer");
+
+// import functions from utils file
+const utils = require("./utils");
```

Comment:
We can destructure this object and just pull out the name of the export we want - `{ writeToFile }`

This means that when we need to use it, we don't have to specify `utils.writeToFile` but can access it directly.